no-computer.md

```diff
@@ -11,41 +11,81 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need met by my idea is the need to play a large range of musical notes which
+can be composed together to make music. I am helping pretty much anyone who would
+want to be able to easily create music and chain notes together to create more complex
+creations. Right now it is difficult to use the entire musical range. Hopefully with
```

Comment:
This is an interesting idea to use the entire musical range. One thing to consider is that most instruments have a fixed range, especially on the lower end of the range (for example, string instruments can play as high as they want, but cannot play lower than their lowest open string). Would it be beneficial to use the entire musical range if no one can ever play the composition? No doubt, it would make wonderful, new music, but in terms of composing it may run into some problems.